src/main/java/frc/trigon/robot/subsystems/turret/TurretCommands.java

```diff
@@ -0,0 +1,17 @@
+package frc.trigon.robot.subsystems.turret;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+
+public class TurretCommands {
+    private static final Turret TURRET = Turret.getInstance();
+
+    public static Command getSetMotorPowerFromPositionCommand(Pose2d robotPosition) {
```

Comment:
You want to have a supplier for the param as the robot position does change. 
And give this a better name. Every command at the end just sets motor power, but why does this one do that? What's the goal? It's like calling the command that sets the target angle position of the arm set target power command.